src/js/components/jtable/TableRow.jsx

```diff
@@ -13,6 +14,7 @@ type Props = {
     className?: string,
     children?: ReactChildren,
     href?: string,
+    linkTo?: string,
```

Comment:
You're right, it doesn't need to wrap, but it can just use the passed-in element in place. I may be wrong, but I don't see `react-router` as a dep in package.json (on master anyways).